examples/multi-gpu-movielens/01-03b-MultiGPU-Download-Convert-ETL-with-NVTabular-Training-with-TensorFlow.ipynb

```diff
@@ -698,110 +710,57 @@
-      "[1,0]<stderr>:2021-04-06 09:25:17.856673: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1261] Device interconnect StreamExecutor with strength 1 edge matrix:\n",
-      "[1,0]<stderr>:2021-04-06 09:25:17.856691: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1267]      0 \n",
-      "[1,0]<stderr>:2021-04-06 09:25:17.856702: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1280] 0:   N \n",
-      "[1,1]<stderr>:2021-04-06 09:25:17.862981: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1406] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 9752 MB memory) -> physical GPU (device: 0, name: Quadro GV100, pci bus id: 0000:2d:00.0, compute capability: 7.0)\n",
-      "[1,0]<stderr>:2021-04-06 09:25:17.863784: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1406] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 9752 MB memory) -> physical GPU (device: 0, name: Quadro GV100, pci bus id: 0000:15:00.0, compute capability: 7.0)\n",
-      "[1,0]<stderr>:2021-04-06 09:25:23.753443: I tensorflow/compiler/mlir/mlir_graph_optimization_pass.cc:116] None of the MLIR optimization passes are enabled (registered 2)\n",
-      "[1,0]<stderr>:2021-04-06 09:25:23.782550: I tensorflow/core/platform/profile_utils/cpu_utils.cc:112] CPU Frequency: 3400000000 Hz\n",
-      "[1,1]<stderr>:2021-04-06 09:25:23.803463: I tensorflow/compiler/mlir/mlir_graph_optimization_pass.cc:116] None of the MLIR optimization passes are enabled (registered 2)\n",
-      "[1,1]<stderr>:2021-04-06 09:25:23.815210: I tensorflow/core/platform/profile_utils/cpu_utils.cc:112] CPU Frequency: 3400000000 Hz\n",
-      "[1,0]<stderr>:2021-04-06 09:25:23.921279: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcublas.so.11\n",
-      "[1,1]<stderr>:2021-04-06 09:25:23.947897: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcublas.so.11\n",
-      "[1,0]<stderr>:2021-04-06 09:25:24.305699: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcublasLt.so.11\n",
-      "[1,1]<stderr>:2021-04-06 09:25:24.348176: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcublasLt.so.11\n",
-      "[1,0]<stdout>:Step #0\tLoss: 0.692326\n",
-      "[1,0]<stdout>:Step #100\tLoss: 0.658852\n",
-      "[1,0]<stdout>:Step #200\tLoss: 0.667175\n",
-      "[1,0]<stdout>:Step #300\tLoss: 0.664376\n",
-      "[1,0]<stdout>:Step #400\tLoss: 0.662460\n",
-      "[1,0]<stdout>:Step #500\tLoss: 0.665670\n",
-      "[1,0]<stdout>:Step #600\tLoss: 0.651120\n",
-      "[1,0]<stdout>:Step #700\tLoss: 0.653099\n",
-      "[1,0]<stdout>:Step #800\tLoss: 0.648519\n",
-      "[1,0]<stdout>:Step #900\tLoss: 0.645759\n",
-      "^C\n"
+      "[1,0]<stderr>:Traceback (most recent call last):\n",
```

Comment:
We arent leaving this error rendered in the notebook are we?

Reply:
I believe this is because of the jupyterlab import problem in the TF container - we'll need to rerender  (https://github.com/NVIDIA/NVTabular/pull/724#discussion_r611243416)

Reply:
I'm trying to re-render but hitting a weird low-level double free from worker 1. :slightly_frowning_face: